src/Kunstmaan/SeoBundle/Twig/SeoTwigExtension.php

```diff
@@ -244,11 +244,16 @@ public function setWebsiteTitle($websiteTitle)
     /**
      * @param $src
      *
-     * @return Seo
+     * @return array|null
      */
     public function getImageDimensions($src)
     {
-        list($width, $height) = getimagesize($src);
+        try {
+            list($width, $height) = getimagesize($src);
+        } catch (\Exception $e) {
+            return null;
+        }
+
```

Comment:
@bakie Sure, that's possible. Not sure about the exact performance of those two functions, but wouldn't that just hit the disk a second time for the same thing `getimagesize` would already do?